### ISSUE TITLE
Handle white and black background separate from regular background layers

### ISSUE THREAD

**Issue by sweco-seqtob** [hajkmap/Hajk#775]
**Is your feature request related to a problem? Please describe.**
Black/white background is today considered to be a background layer but the logic doesn't fit the radio-button setup (BackgroundSwitcher).

**Describe the solution you'd like**
There should be an extra option to choose "No backgroundlayer" and the option to change black/white background should 
be managed separately. 

This would make it possible to have only a white or black background but also to put backgroundlayers on top of the black or white background.

**Comment by jesade-vbg** [comment]
Could you provide a mockup of your idea so it's easier to visualize?

**Comment by jacobwod** [comment]
I think we can imagine this as some On/Off switch, perhaps at top of the background layers section:
```
---
x/- Map background color (light / dark)
o     Orto 2019
-     Orto 2018
-     Orto 2017
---

**Comment by jesade-vbg** [comment]
Is this a complete representation?
Am I understanding correctly?

**UI**
```
------------------------------
( * ) white background (white is default if light mode)
( * ) black background (black is default if dark mode)
------------------------------
( * ) No background layer. (option is optional in config)
( * ) Topo 1
( * ) Topo 2
( * ) Orto 1
( * ) Orto 2
------------------------------
```

if white is the only option, white is default background option.
if black is the only option, black is default background option.
if both is visible, default is light>white or dark Mode>black.
if no choice is visible.............................

**Comment by sweco-seqtob** [comment]
I think black/white should be radio buttons to make it impossible to have both enabled at the same time.

I think dark-mode is handled automatically by MaterialUI and should not be related to white or black background (@Hallbergs you had more information regarding this?)

You choose your "canvas", represented by choosing black/white, to paint your background layer at

Does that make sense?

**Comment by Hallbergs** [comment]
The color you see if you haven't enabled either white or black is the  background of the application (which depends on dark or light theme). I guess the map canvas is transparent until we set it to white or black - and i like the idea with radiobuttons for those options.

**Comment by jacobwod** [comment]
I agree with all the above. One minor addition is that we could add an option that resets to system default (i.e. prior user clicks white/black), which would then unset (is it even possible) the background color on canvas.
```
------------------------------
( x ) System default (controlled by theme)
( * ) White background 
( * ) Black background 
( * ) Custom color HEX: _________ 
------------------------------
( * ) No background layer. (option is optional in config)
( * ) Topo 1
( * ) Topo 2
( * ) Orto 1
( * ) Orto 2
------------------------------
```

**Comment by Hallbergs** [comment]
Custom color would be nice! Could a color picker be a good idea or is it too much?

**Comment by jesade-vbg** [comment]
The last mockup is great, but I'm a bit worried about loosing focus from the actual backgrounds. We also have mobile views to keep in mind (scroll?). Custom color is a great addition and an expandable picker too, but I'm not sure if these options will be used much, and it becomes a bit bloated. As long as it's optional it's fine.

**Comment by jacobwod** [comment]
It could be hidden by default in a collapsed panel ("accordion").

**Comment by Hallbergs** [comment]
Let's make sure to add something in the lines of the discussion above to the new LayerSwitcher-plugin.
